benchmarks/ModelingToolkit/ThermalFluid.jmd

```diff
@@ -278,6 +282,33 @@ function build_run_problem(fsys, N; target=JuliaSimCompiler.JuliaTarget())
   t_ss, t_fode, t_run
 end
 
+function test_speed(fsys, N; solver=FBDF(), target=JuliaSimCompiler.JuliaTarget())
+    tspan = (0.0, 19*3600)
+    t_total = @elapsed begin
+        @named testbench = TestBenchPreinsulated(L=470, N=N, dn=0.3127, t_layer=[0.0056, 0.058])
+        sys = structural_simplify(fsys(testbench))
+        if target === JuliaSimCompiler.JuliaTarget()
+            prob = ODEProblem(sys, [], tspan, sparse=true)
+        else
+            prob = ODEProblem(sys, target, [], tspan, sparse=true)
+        end
+        prob.u0 .= 12.0
+        solve(prob, solver, reltol=1e-6, abstol=1e-6, saveat=100);
+    end
+end
+```
+
+```julia
+N_x = [5, 10, 20, 40, 60, 80, 160, 320, 480, 640, 800, 960, 1280];
```

Comment:
```suggestion
N_x = [5, 10, 20, 40, 60, 80, 160];
```

Reply:
On my desktop, MTK's `structural_simplify` took more than 10 minutes for the largest size.

Reply:
Should we do a cutoff on pure MTK?

Reply:
Yeah, I think that's a good idea, depending on how well OpenModelica can handle larger sizes/finer discritizations.

It's nice to go further for the faster methods to make claims like "with fixed time, here is how many discretized components you can use".

Reply:
okay, I got this all working except the OpenModelica part now.